Automattic/wp-calypso
# Migrate vertical nav and vertical nav item styles

#### Changes proposed in this Pull Request

Migrate styles for vertical nav and nav items.

#### Testing instructions

Go to http://calypso.localhost:3000/devdocs/design/vertical-nav and confirm that Vertical Nav and its items are styled correctly.

##### Unstyled
<img width="1333" alt="screen shot 2018-10-03 at 12 09 44 am" src="https://user-images.githubusercontent.com/10561050/46372394-9c684d80-c6bd-11e8-9e62-04c57c387167.png">

##### Styled
<img width="1319" alt="screen shot 2018-10-03 at 12 10 44 am" src="https://user-images.githubusercontent.com/10561050/46372399-9ffbd480-c6bd-11e8-90d2-4a6c2d7fe47a.png">

#27515
